Apps/WebClient/src/ClientApp/src/utility/dateUtil.ts

```diff
@@ -0,0 +1,22 @@
+export default abstract class DateUtil {
```

Comment:
moment does a good job of handling all this too

Reply:
yeah, ideally this would be gone too. We can remove this with proper moment fucntions